docs/beamrunner/beamrunner-2-install.md

```diff
@@ -0,0 +1,128 @@
+
+To create a Streaming Analytics service:
+
+1. On the [Bluemix catalog](https://console.ng.bluemix.net/catalog/services/streaming-analytics/?cm_mc_uid=05407033353914938482142&amp;cm_mc_sid_50200000=) page, log in. If you don't yet have a Bluemix account, you can create one.
+2. On the Bluemix dashboard page, click **Catalog**.
+3. Use the **Filter** option to search for Streaming Analytics.
+4. Click the **Streaming Analytics** service to configure an instance.
+5. On the catalog page for the Streaming Analytics service, change **Service name** to something meaningful to you, for example, `Streaming Analytics-beam`.
+6. Click **Create**. The service page opens and your service starts automatically. The service name appears as the title of the service page.
+
+For more information about the Streaming Analytics service, see [Introduction to the Bluemix Streaming Analytics Service](https://developer.ibm.com/streamsdev/docs/streaming-analytics-now-available-bluemix-2/).
+
+## Downloading and configuring Streams Runner
+
+1. Launch the Streams Console:
+    1. Navigate to the **Manage** tab of your Streaming Analytics Service.
+    1. Click **Launch**.
+1. Download Streams Runner:
+    1. In the Streams Console, click **Help > Download > Download Streams Runner for Apache Beam**.
+    1. Select **Save File**, specify a location if necessary, and click **OK**.
+1. Extract the toolkit by entering the following command where you downloaded the file:
+```
+tar -zxvf com.ibm.streams.beam-2.0.0-beta.tar.gz
+```
+1. (Optional) Configure the environment variables. Although the variables are not required, the documentation refers to them for convenience. If you do not set the environment variables, you must use the full paths when you run the sample applications.
+
+| Name | Description | Notes |
+  | --- | --- | --- |
+  | STREAMS\_INSTALL | The path to the IBM Streams installation | Set by sourcing the `bin/streamsprofile.sh` file. This variable is already set in the Quick Start Edition. |
+  | STREAMS\_RUNNER\_HOME | The path to the extraction location of the `com.ibm.streams.beam.tar.gz` file | Must be set by the user by using the `export` command. |
+  | STREAMS\_BEAM\_TOOLKIT | The path to the Streams Runner toolkit (`$STREAMS\_RUNNER\_HOME/com.ibm.streams.beam`) | Set by using the `export` command. |
+  | VCAP\_SERVICES | The path to the Bluemix credentials file. For more information about the credentials file, see [Creating a credentials file for your Streaming Analytics service](#creating-a-credentials-file-for-your-streaming-analytics-service). | Set by using the `export` command. |
+  | STREAMING\_ANALYTICS\_SERVICE\_NAME | The path to the Bluemix credentials file. For more information about the credentials file, see [Creating a credentials file for your Streaming Analytics service](#creating-a-credentials-file-for-your-streaming-analytics-service). | Set by using the `export` command. |
+
+## Validating the Streams Runner installation

```

Comment:
@mrshenli  Maybe it is too late for this release, but Would it be easier to provide a hash (md5/sha1) with the download instead of asking them to manually verify the folder structure?